pkgs/applications/office/treesheets/default.nix

```diff
@@ -7,17 +7,17 @@
 , makeWrapper
 , wxGTK
 , Cocoa
-, unstableGitUpdater
+, gitUpdater
 }:
 
 stdenv.mkDerivation rec {
   pname = "treesheets";
-  version = "unstable-2024-01-03";
+  version = "7398648179";
```

Comment:
Copying from the commit description:

  For all intents and purposes, that seems to be their versioning now.

  - Last semver release 1.0.2 was more than 4 years ago
  - That's the version shown in the About dialogue
  - They previously said that they »do "releases" super infrequently or not at all«
  - Basically every commit is tagged like this, numbers are always-increasing

Due to that, I see no reason to use the `unstableGitUpdater` on this.